exchanges/bitstamp/bitstamp_test.go

```diff
@@ -500,174 +516,165 @@ func TestSubmitOrder(t *testing.T) {
+		assert.WithinRange(t, o.Date, time.Now().Add(-24*time.Hour), time.Now(), "Date should be correct")
 	}
 }
 
 func TestCancelExchangeOrder(t *testing.T) {
 	t.Parallel()
 
 	if !mockTests {
-		sharedtestvalues.SkipTestIfCannotManipulateOrders(t, b, canManipulateRealOrders)
-	}
-
-	orderCancellation := &order.Cancel{
-		OrderID:   "1234",
-		AssetType: asset.Spot,
+		sharedtestvalues.SkipTestIfCredentialsUnset(t, b, canManipulateRealOrders)
 	}
-	err := b.CancelOrder(context.Background(), orderCancellation)
-	switch {
-	case !sharedtestvalues.AreAPICredentialsSet(b) && err == nil && !mockTests:
-		t.Error("Expecting an error when no keys are set")
-	case sharedtestvalues.AreAPICredentialsSet(b) && err != nil && !mockTests:
-		t.Errorf("Could not cancel orders: %v", err)
-	case mockTests && err == nil:
-		t.Error("Expecting an error until QA pass is completed")
+	err := b.CancelOrder(context.Background(), &order.Cancel{
+		OrderID: "1453282316578816",
+	})
+	if !mockTests {
+		assert.ErrorContains(t, err, "Order not found")
+	} else {
+		assert.NoError(t, err, "CancelExchangeOrder should not error")
 	}
 }
 
 func TestCancelAllExchangeOrders(t *testing.T) {
 	t.Parallel()
 
 	if !mockTests {
-		sharedtestvalues.SkipTestIfCannotManipulateOrders(t, b, canManipulateRealOrders)
+		sharedtestvalues.SkipTestIfCredentialsUnset(t, b, canManipulateRealOrders)
 	}
-
 	resp, err := b.CancelAllOrders(context.Background(),
 		&order.Cancel{AssetType: asset.Spot})
-	switch {
-	case !sharedtestvalues.AreAPICredentialsSet(b) && err == nil && !mockTests:
-		t.Error("Expecting an error when no keys are set")
-	case sharedtestvalues.AreAPICredentialsSet(b) && err != nil && !mockTests:
-		t.Errorf("Could not cancel orders: %v", err)
-	case mockTests && err != nil:
-		t.Errorf("Could not cancel orders: %v", err)
-	}
-
+	assert.NoError(t, err, "TestCancelAllExchangeOrders should not error")
 	if len(resp.Status) > 0 {
 		t.Errorf("%v orders failed to cancel", len(resp.Status))
 	}
 }
 
 func TestModifyOrder(t *testing.T) {
 	t.Parallel()
-	sharedtestvalues.SkipTestIfCannotManipulateOrders(t, b, canManipulateRealOrders)
-	_, err := b.ModifyOrder(context.Background(), &order.Modify{AssetType: asset.Spot})
-	if err == nil {
-		t.Error("ModifyOrder() Expected error")
+
+	if !mockTests {
+		sharedtestvalues.SkipTestIfCredentialsUnset(t, b, canManipulateRealOrders)
 	}
+	_, err := b.ModifyOrder(context.Background(), &order.Modify{AssetType: asset.Spot})
+	assert.ErrorIs(t, err, common.ErrFunctionNotSupported)
 }
 
 func TestWithdraw(t *testing.T) {
 	t.Parallel()
 
 	if !mockTests {
-		sharedtestvalues.SkipTestIfCannotManipulateOrders(t, b, canManipulateRealOrders)
+		sharedtestvalues.SkipTestIfCredentialsUnset(t, b, canManipulateRealOrders)
 	}
-
-	withdrawCryptoRequest := withdraw.Request{
+	w, err := b.WithdrawCryptocurrencyFunds(context.Background(), &withdraw.Request{
 		Exchange:    b.Name,
-		Amount:      -1,
+		Amount:      6,
```

Comment:
If someone were to enable live tests + set `canManipulatRealOrders` to true with their API keys set, run `go test ./...`  or using their IDE's "test package" functionality, this would accidently send funds to the donation address which would be nice, but not good. Live tests with anything financial were intentionally disarmed because of this

Reply:
I have changed it back to -1

Reply:
I realised that it's meaningless to run a live test with a negative since it never hits their server. So let's just skip the test if live.